doc/how-to/vshard_quick.rst

```diff
@@ -1,160 +1,491 @@
         unknown: 0
-        available_rw: 3000
+        available_rw: 1000
       status: 0
       alerts: []
     ...
 
-..  _vshard-config-cluster-example:
-
-Sample configuration
---------------------
-
-The configuration of a simple sharded cluster can look like this:
-
-..  code-block:: kconfig
-
-    local cfg = {
-        memtx_memory = 100 * 1024 * 1024,
-        bucket_count = 10000,
-        rebalancer_disbalance_threshold = 10,
-        rebalancer_max_receiving = 100,
-        sharding = {
-            ['cbf06940-0790-498b-948d-042b62cf3d29'] = {
-                replicas = {
-                    ['8a274925-a26d-47fc-9e1b-af88ce939412'] = {
-                        uri = 'storage:storage@127.0.0.1:3301',
-                        name = 'storage_1_a',
-                        master = true
-                    },
-                    ['3de2e3e1-9ebe-4d0d-abb1-26d301b84633'] = {
-                        uri = 'storage:storage@127.0.0.1:3302',
-                        name = 'storage_1_b'
-                    }
-                },
-            },
-            ['ac522f65-aa94-4134-9f64-51ee384f1a54'] = {
-                replicas = {
-                    ['1e02ae8a-afc0-4e91-ba34-843a356b8ed7'] = {
-                        uri = 'storage:storage@127.0.0.1:3303',
-                        name = 'storage_2_a',
-                        master = true
-                    },
-                    ['001688c3-66f8-4a31-8e19-036c17d489c2'] = {
-                        uri = 'storage:storage@127.0.0.1:3304',
-                        name = 'storage_2_b'
-                    }
-                },
-            },
-        },
-    }
-
-This cluster includes one ``router`` instance and two ``storage`` instances.
-Each ``storage`` instance includes one master and one replica.
-The ``sharding`` field defines the logical topology of a sharded Tarantool cluster.
-All the other fields are passed to ``box.cfg()`` as they are, without modifications.
-See the :ref:`Configuration reference <vshard-config-reference>` section for details.
-
-On routers, call ``vshard.router.cfg(cfg)``:
-
-..  code-block:: lua
-
-    cfg.listen = 3300
-
-    -- Start the database with sharding
-    vshard = require('vshard')
-    vshard.router.cfg(cfg)
-
-On storages, call ``vshard.storage.cfg(cfg, instance_uuid)``:
-
-..  code-block:: lua
-
-    -- Get instance name
-    local MY_UUID = "de0ea826-e71d-4a82-bbf3-b04a6413e417"
-
-    -- Call a configuration provider
-    local cfg = require('localcfg')
-
-    -- Start the database with sharding
-    vshard = require('vshard')
-    vshard.storage.cfg(cfg, MY_UUID)
-
-``vshard.storage.cfg()`` automatically calls ``box.cfg()`` and configures the listen
-port and replication parameters.
-
-For a sample configuration, see ``router.lua`` and ``storage.lua`` in the
-``example/`` directory of the `vshard repository <https://github.com/tarantool/vshard>`__.
+
+.. _vshard-quick-start-working-adding-data:
+
+Adding data
```

Comment:
Shouldn't we add a last section like 'access data', where we show an example of put/get requests to the router? It is OK to perform it from a console connected to the router.

Reply:
Yep, this is absolutely necessary. Totally forgot about this, will do.

Reply:
Two sections instead of `Adding data`:
- [Adding and selecting data](https://docs.d.tarantool.io/en/doc/sharding-get-started/how-to/vshard_quick/#adding-and-selecting-data)
- [Checking data distribution](https://docs.d.tarantool.io/en/doc/sharding-get-started/how-to/vshard_quick/#checking-data-distribution)

Reply:
Thanks for the update!